### PR TITLE
[Calendar Event Type] add youtube to calendar event type

### PR DESCRIPTION
**Pull request by tathagat2006**
Signed-off-by: Tathagat Thapliyal <tathagat.thapliyal@gmail.com>

### CODE REVIEW

**Comment by tathagat2006** [comment]
image

File: api/graphql/query/CalendarEvent/type.graphql [hunk 1]
```diff
@@ -6,6 +6,7 @@ enum CalendarEventType {
   Meeting
   OOO
   Other
+  Youtube
```
Comment (0xnmn) [inline api/graphql/query/CalendarEvent/type.graphql:8]: Move this above other

File: api/graphql/query/eventTypeColors.js [hunk 2]
```diff
@@ -27,4 +27,8 @@ export default () => [
     name: 'Other',
     color: '#d4d4d4',
   },
+  {
```
Comment (0xnmn) [inline api/graphql/query/eventTypeColors.js]: Move this above other

**Review by 0xnmn — CHANGES_REQUESTED** [review]

File: api/graphql/query/CalendarEvent/type.graphql (hunk 1 above)
Comment (tathagat2006) [reply api/graphql/query/CalendarEvent/type.graphql:8]: Updated

File: api/graphql/query/eventTypeColors.js (hunk 2 above)
Comment (tathagat2006) [reply api/graphql/query/eventTypeColors.js]: Updated

**Review by 0xnmn — APPROVED** [review]

**Pull request merged**